src/Validator.php

```diff
@@ -117,28 +113,20 @@ public function if(array $conditions, array $then_hash, array $else_hash = []):
      *
      * @return array<string, string> array of errors in format: [field_name => error_message]
      */
-    public function validate(Model $model, ?string $intent = null): array
+    public function validate(Model $model): array
     {
         // initialize Validator, set data
-        $v = new \Valitron\Validator($model->get());
+        $v = new ValitronValidator($model->get());
```

Comment:
```suggestion
        $validator = new ValitronValidator($model->get());
```

`$v` might easily imply "value"